libraries/AP_Notify/RGBLed.h

```diff
@@ -38,7 +38,12 @@ class RGBLed: public NotifyDevice {
 
     // handle LED control, only used when LED_OVERRIDE=1
     virtual void handle_led_control(const mavlink_message_t &msg) override;
-    
+
+    // RGB override
+    // override other all methods for given duration (0 = for ever)
+    // used with scripting
+    virtual void rgb_override(uint8_t r, uint8_t g, uint8_t b, uint8_t rate_hz, uint16_t duration) override;
```

Comment:
it would be good to give the units for duration

Reply:
fixed, added to the header and also changed the param to duration_ms